### ISSUE TITLE
Can we use sympy's plotting routines for graphics? 

### ISSUE THREAD

**Issue by rocky** [mathics/Mathics#909]
Sympy's plotting routines blow away what is in Mathcs. See https://docs.sympy.org/latest/modules/plotting.html

Now that we have a recent version of sympy, maybe we can chop out a *lot* of code and reduce maintanance by simply converting the Mathematica parameters into the equivalent Sympy paraemters and letting Sympy do all of the heavy lifting. 

One thing that really annoys me about Mathics plots that doesn't appear in Sympy plots (as seen in the link) or Mathematica is the labelling of axes.

**Comment by mmatera** [comment]
Indeed, and also uses matplotlib, what could be useful for the iwolfram / imathics interface. Cool!Also we have inside mathics an interface to asymptote, which also is currently quite powerful...

**Comment by GarkGarcia** [comment]
I think that an awesome idea, but we should wait to after the 1.1 release.

**Comment by rocky** [comment]
> I think that an awesome idea, but we should wait to after the 1.1 release.

That is why I added it to https://github.com/mathics/Mathics/issues/912 which should probably get turned into milestones.

**Comment by rocky** [comment]
Oops correctrion. I had _intended_ to do so. It is there now though.
